crates/spawner/src/spawner.rs

```diff
@@ -17,10 +16,21 @@ pub(crate) struct SpawnerPlugin;
 
 impl Plugin for SpawnerPlugin {
     fn build(&self, app: &mut App) {
-        app.add_systems(Update, spawn.run_if(in_state(GameState::Playing)));
+        app.add_systems(
+            Update,
+            spawn
+                .run_if(in_state(GameState::Playing))
+                .in_set(SpawnerSet::Spawn),
+        );
     }
 }
 
+#[derive(SystemSet, Debug, Clone, PartialEq, Eq, Hash)]
+pub enum SpawnerSet {
+    /// This exists so that we can manage newly spawned objects by `.after(SpawnerSet::Spawn)`.
```

Comment:
~I would rather document what is done in the set than what should be done after it (possibly in different creates).~

It is not needed at all, newly spawned entities are not available during the same (`Update`) schedule. Spawn commands are executed only between schedules (e.g. between `Update` and `PostUpdate`).